app/decorators/work_show_decorator.rb

```diff
@@ -101,12 +106,7 @@ def public_collections
   # list, because no reason to duplicate it right after the representative.
   def member_list_for_display
     @member_list_display ||= begin
-      members = model.members.
-        with_representative_derivatives.
-        where(published: true).
-        order(:position).
-        to_a
-
+      members = ordered_public_members
       members.delete_at(0) if members[0] == representative_member
```

Comment:
Good call. In practice it's not a big deal because member_list_for_display is itself memoized, so it's impossible to run the core of its code more than once. But still, oops. I'm adding a .dup . Yes, ordered_public members is used in a couple places.